demo/primefaces-users-portlet/src/main/java/com/liferay/faces/demos/service/CompanyLocalServiceTracker.java

```diff
@@ -25,7 +25,28 @@
  */
 public class CompanyLocalServiceTracker extends ServiceTracker<CompanyLocalService, CompanyLocalService> {
 
-	public CompanyLocalServiceTracker(BundleContext bundleContext) {
+	private BundleContext _bundleContext;
+	private CompanyLocalService _companyLocalService;
+	
+
+	public CompanyLocalServiceTracker(BundleContext bundleContext) {		
 		super(bundleContext, CompanyLocalService.class, null);
+		_bundleContext = bundleContext;
+	}
+
+	@Override
+	public CompanyLocalService addingService(ServiceReference<CompanyLocalService> reference) {
+		_companyLocalService = _bundleContext.getService(reference);
+		return _companyLocalService;
+	}
```

Comment:
@juangon This is exactly what `super.addingService` does -- I don't see how this provides additional functionality?